.github/workflows/publish-apps-release.yml

```diff
@@ -123,7 +123,7 @@ jobs:
           path: out
           glob: '*'
           parent: false
-          destination: 'dl.kittycad.io/releases/modeling-app/test/new-workflow'
+          destination: 'dl.kittycad.io/releases/modeling-app'
```

Comment:
https://github.com/KittyCAD/modeling-app/actions/runs/12038143886/job/33563153388 went perfectly but it pushed builds to the testing location